### DISCUSSION TITLE
Support for Graphical Applications

### DISCUSSION THREAD

**Discussion by TheStaticBits** [community/community#17137] [Codespaces] [answered]
Support for running applications in Codespaces that create a window, such as SDL2 or Pygame, if possible, would be useful.

**Comment by retronav** [comment, marked as answer]
Hi, GitHub Codespaces supports a [lightweight Fluxbox-based desktop](https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/desktop-lite.md) that might solve your problems of running a GUI. The desktop can be accessed via a VNC Viewer Client or noVNC running on port 6080.

**Reply by josealvarez97** [reply]
Do you think something like that could be used to develop for Unity3D? @retronav

**Reply by retronav** [reply]
I think you can. I haven't had exposure to Unity3D but looking at it, it'll probably run fine.
